### PR TITLE
Doc: Perlmutter CUDA 11.7 Update

### PR DESCRIPTION
**Pull request by ax3l**
Update the documentation for FBPIC installation on Perlmutter.

### CODE REVIEW

File: docs/source/install/install_perlmutter.rst [hunk 1]
```diff
@@ -34,15 +51,14 @@ Installation of FBPIC and its dependencies
 
    ::
 
-       conda install -c conda-forge python numba scipy h5py mkl cudatoolkit=11.5
-       pip install cupy-cuda115
-       MPICC="cc -shared -target-accel=nvidia80" pip install --force --no-cache-dir --no-binary=mpi4py mpi4py
+       conda install -c conda-forge python numba scipy h5py mkl cudatoolkit=11.7 cupy
+       MPICC="cc -shared -target-accel=nvidia80" python3 -m pip install --force --no-cache-dir --no-binary=mpi4py mpi4py
```
Comment (ax3l) [inline docs/source/install/install_perlmutter.rst:46]: This existing part is pretty risky, mixing the system compilers for MPI4PY with the ones for the rest of the software stack.
If it works I will not change it, but I recommend against doing this :)

File: docs/source/install/install_perlmutter.rst [hunk 2]
```diff
@@ -25,7 +25,23 @@ Setting up Anaconda
...
-        source activate fbpic
+        conda activate fbpic
+
+- Configure your conda to use the faster `libmamba` `dependency solver <https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community>`__.
+
+   .. code-block:: bash
+
+      conda install -n fbpic conda
+      conda install -n fbpic conda-libmamba-solver
+      conda config --set solver libmamba
```
Comment (ax3l) [inline docs/source/install/install_perlmutter.rst]: This does not yet work...

Comment (ax3l) [reply docs/source/install/install_perlmutter.rst]: removed for now

File: docs/source/install/install_perlmutter.rst (hunk 1 above)
Comment (ax3l) [reply docs/source/install/install_perlmutter.rst:46]: Still works :D

Comment (ax3l) [reply docs/source/install/install_perlmutter.rst:46]: I will pretend I did not see this.

**Review by RemiLehe — APPROVED** [review]

**Pull request merged**